homeassistant/components/recorder/__init__.py

```diff
@@ -570,6 +577,11 @@ def async_hourly_statistics(self, now):
         start = statistics.get_start_time()
         self.queue.put(StatisticsTask(start))
 
+    @callback
+    def async_update_statistics_metadata(self, statistic_id, entity_id):
+        """Update statistics metadata for a statistic_id."""
+        self.queue.put(UpdateStatisticsMetadataTask(statistic_id, entity_id))
```

Comment:
I'm confused, `UpdateStatisticsMetadataTask` takes a `statistic_id` and `unit_of_measurement`. Not an `entity_id` ?

Reply:
Fixed in 0a49db8